functions.js

```diff
@@ -0,0 +1,38 @@
+let addIngredient = () => {
+    let newIngredient = document.getElementById("ingredient_input").value;
+    var node = document.createElement('li');
+    node.appendChild(document.createTextNode(newIngredient));
+    document.getElementById("ingredient_list").appendChild(node);
+    document.getElementById("ingredient_input").value = "";
+}
+
+let addStep = () => {
+    let newStep = document.getElementById("step_input").value;
+    var node = document.createElement('li');
```

Comment:
What's the reason for using let and var? In general, most JS developers recommend always using let instead of var. Here's the difference between the two: https://stackoverflow.com/a/11444416